Model/Classes/FlightClass.cs

```diff
@@ -0,0 +1,10 @@
+﻿using AirportTicketBooking.Utils;
+
+namespace AirportTicketBooking.Model.Classes;
+
+public class FlightClass
+{
+    public FlightClassType Type { get; set; }
+    public int Seats { get; set; }
```

Comment:
NIT: can we rename this or at least update ToString of Flight class since this name does not have the same meaning for  empty seats